src/root/views/Search/ProjectTable/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import LanguageContext from '#root/languageContext';
+import Container from '#components/Container';
+import {
+  createDateColumn,
+  createNumberColumn,
+  createStringColumn,
+} from '#components/Table/predefinedColumns';
+import Table from '#components/Table';
+
+import { Project } from '../index';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  className?: string;
+  data: Project[] | undefined;
+}
+
+function ProjectKeySelector(project: Project) {
+  return project.id;
+}
+
+function ProjectTable(props: Props) {
+  const {
+    className,
+    data,
+  } = props;
+
+  const { strings } = React.useContext(LanguageContext);
+
+  const columns = [
+    createStringColumn<Project, number>(
+      'name',
+      'Emergency',
+      (project) => project.name,
+    ),
+    createStringColumn<Project, number>(
+      'national_society',
+      'National Society/ ERU',
+      (project) => project.national_society,
+    ),
+    createStringColumn<Project, number>(
+      'name',
+      'Project/ Activity Name',
+      (project) => project.name,
+    ),
+    createDateColumn<Project, number>(
+      'start_date',
+      'Start-End Dates',
+      (project) => project.start_date,
+    ),
+    createStringColumn<Project, number>(
+      'regions',
+      'Provinces/Region',
+      (project) => project.regions[0],
```

Comment:
If there are multiple regions involved, it's better to concatenate them and show all of them.